docs/source/index.md

```diff
@@ -1,4 +1,25 @@
-# The Jupyter Notebook
+# Jupyter Notebook Documentation
+
+Welcome to the **Jupyter Notebook** documentation site. **Jupyter Notebook**
+is a lightweight notebook authoring application, and is a part of [Project
+Jupyter](https://docs.jupyter.org/en/latest/), a large umbrella project
+centered around the goal of providing tools for interactive computing
+with [computational notebooks](https://docs.jupyter.org/en/latest/#what-is-a-notebook).
+
+A [computational notebook](https://docs.jupyter.org/en/latest/#what-is-a-notebook)
+is a shareable document that combines computer
+code, plain language descriptions, data, rich visualizations like 3D models,
+charts, graphs and figures, and interactive controls. A notebook, along with
+an editor like **Jupyter Notebook**, provides a fast interactive environment for
+prototyping and explaining code, exploring and visualizing data, and sharing
+ideas with others.
+
+**Jupyter Notebook** is a sibling to other notebook authoring applications under
```

Comment:
Similar comment to https://github.com/jupyterlab/jupyterlab/pull/14568: 

If we mention "other notebook authoring applications under the Project Jupyter umbrella" in plural, it would be nice to name some other applications on top of the JupyterLab and provide links to them. 

After thinking about it, other than JupyterLab and Notebook I can't name "other notebook authoring applications under the Project Jupyter umbrella". For example, Voila and Nbcovert are more of converters and Jupyter Book is a book building application.

Reply:
Jupyter Desktop and JupyterLite come to mind for me, though I think we should keep additional mentions to a minimum (maybe 1 or 2 extras) to not over complicate this part of the text.

Reply:
It would be nice to mention 1 or 2 extras of Notebook authoring applications.